src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/NuGet.PackageManagement.VisualStudio.csproj

```diff
@@ -45,7 +45,7 @@
       <EmbedInteropTypes>True</EmbedInteropTypes>
     </Reference>
     <Reference Include="VSLangProj150">
-      <HintPath>$(SolutionPackagesFolder)VSLangProj150.1.0.0\lib\net46\VSLangProj150.dll</HintPath>
+      <HintPath>$(SolutionPackagesFolder)VSLangProj150.15.0.26229\lib\net46\VSLangProj150.dll</HintPath>
```

Comment:
The weird thing is that the assembly using during compilation is from the PublicAssemblies folder. 

I need to investigate how/why that one is being picked despite an explicit reference to this assembly.

Reply:
fixed.